source/source_integration_test.go

```diff
@@ -0,0 +1,490 @@
+// Copyright © 2022 Meroxa, Inc. & Yalantis
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package source
+
+import (
+	"context"
+	"crypto/rand"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/conduitio-labs/conduit-connector-redshift/config"
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"github.com/jmoiron/sqlx"
+	"github.com/matryer/is"
+)
+
+const envNameDSN = "REDSHIFT_DSN"
+
+func TestSource_Read_noTable(t *testing.T) {
```

Comment:
noTable is it means we don't set it on config?

Reply:
No, it meant the table has no data.
But I've updated the names of test functions.